internal/cmd/commands/server/worker_initial_upstreams_reload_test.go

```diff
@@ -0,0 +1,128 @@
+//go:build !hsm
+// +build !hsm
```

Comment:
I don't think we need this second line given our minimum Go version?

Reply:
Yep, fixed. At some point we should probably remove the remaining `+build` lines like the one I copied this from.